packages/server/src/fhir/routes.ts

```diff
@@ -314,17 +314,11 @@ protectedRoutes.use(
         graphqlMaxDepth: ctx.project.systemSetting?.find((s) => s.name === 'graphqlMaxDepth')?.valueInteger,
         graphqlMaxPageSize: ctx.project.systemSetting?.find((s) => s.name === 'graphqlMaxPageSize')?.valueInteger,
         graphqlMaxSearches: ctx.project.systemSetting?.find((s) => s.name === 'graphqlMaxSearches')?.valueInteger,
+        searchOnReader: ctx.project.systemSetting?.find((s) => s.name === 'searchOnReader')?.valueBoolean,
         transactions: ctx.project.features?.includes('transaction-bundles'),
       },
     };
 
-    if (request.url.includes('$graphql')) {
```

Comment:
Does this mean that GraphQL searches will no longer go to the reader unless configured to at the project level?

Reply:
Oh I see, it just happens in `graphql.ts` now. Nevermind